test/scripts/curl

```diff
@@ -46,6 +46,14 @@ for (const [re, fn] of /** @type {const} */([
 
         return require(`../data/${token.symbol}/getBalanceOfToken_${accountId}.json`);
     }],
+    [/^tokens\/([\w.]+)\/nfts\/(\d+)$/, (tokenId, nftNumber) => {
+        assert(typeof tokenId === 'string');
+        const token = tokens[tokenId];
+        if (token === undefined)
+            return { _status: { messages: [{ message: 'Not found' }] } };
```

Comment:
We should return `undefined` here, so the main loop then return this not found response.